### PR TITLE
convert to nwb

### PR DESCRIPTION
**Pull request by bendichter**

### CODE REVIEW

File: element_animal/export/nwb.py [hunk 1]
```diff
@@ -5,22 +5,23 @@
...
+        ),
         description=json.dumps(subject_info, default=str),
-        genotype=' x '.join((subject.Line.Allele * subject.Subject.Line
-                             & subject_key
-                             ).fetch('allele')),
-        species=subject_info['species']
+        genotype=" x ".join(
+            (subject.Line.Allele * subject.Subject.Line & subject_query).fetch("allele")
+        ),
+        #species=subject_info["species"],
```
Comment (CBroz1) [inline element_animal/export/nwb.py:26]: Why not include species here? We could add a conditional to only include if not null. Or are you more concerned with the lack of enforcement around latin naming, preferring to exclude if we can't verify?

**Review by CBroz1 — APPROVED** [review]

File: element_animal/export/nwb.py (hunk 1 above)
Comment (kabilar) [inline element_animal/export/nwb.py:26]:
```suggestion
```

Comment (kabilar) [reply element_animal/export/nwb.py:26]: We can remove for now and add back in a future update.

**Review by kabilar — APPROVED** [review]
Thanks @bendichter.  Minor suggestion above.

**Comment by bendichter** [comment]
Thanks @kabilar! Please note that DANDI will soon be requiring NWB files to include species information.

Also, I do not have merge privileges so go ahead and merge yourself when you are ready.

**Review by kabilar — APPROVED** [review]

**Comment by kabilar** [comment]
> Thanks @kabilar! Please note that DANDI will soon be requiring NWB files to include species information.

Good to know.  Thanks @bendichter!

**Pull request merged**